official/resnet/ctl/ctl_imagenet_main.py

```diff
@@ -215,70 +228,44 @@ def step_fn(inputs):
       else:
         step_fn(test_ds_inputs)
 
-    if flags_obj.enable_function:
+    if flags_obj.use_tf_function:
       train_step = tf.function(train_step)
       test_step = tf.function(test_step)
 
-    epoch_exp_per_sec = []
-    stats = {}
+    time_callback.on_train_begin()
     for epoch in range(train_epochs):
-      logging.info('Starting to run epoch: %s', epoch)
       train_iterator = iter(train_ds)
 
       step = 0
       total_loss = 0.0
-      batch_exp_per_sec = []
       for step in range(train_steps):
-        start_time = time.time()
-        learning_rate = compute_learning_rate(
-            epoch + 1 + (float(step) / steps_per_epoch))
-        optimizer.lr = learning_rate
-        start_time = time.time()
-        total_loss += train_step(next(train_iterator))
-        end_time = time.time()
-        elapsed_time = end_time - start_time
-        samples_per_sec = flags_obj.batch_size / elapsed_time
-        # We skip the first step for warmup purposes. We can
-        # add a flag for tuning this.
-        if step > 0:
-          batch_exp_per_sec.append(samples_per_sec)
+        training_accuracy.reset_states()
```

Comment:
hm, do you want to reset accuracy every step? usually it's at end of every epoch

Reply:
Wrong indentation during refactor. Done.